### ISSUE TITLE
How to build on Illumos/OpenIndiana?

### ISSUE THREAD

**Issue by hoper-a11y** [copperspice/copperspice#147]
Could you provide instruction? All of the current instructions are for Linux based systems.

**Comment by hoper-a11y** [comment]
Sorry. After I checked the CMakeLists.txt file I found CopperSpice doesn't support Illumos system. I will close this issue.

**Comment by janwilmans** [comment]
While you're correct and we don't explicitly support OpenIndiana, it could probably work, why not try it. I'm willing to help if needed.

**Comment by hoper-a11y** [comment]
> While you're correct and we don't explicitly support OpenIndiana, it could probably work, why not try it. I'm willing to help if needed.

I'm already tried. I found copperspice has hard dependency on alsa. Even when I disable multimedia support cmake still lists alsa as not found dependency. On OpenIndiana, we don't have alsa. cmake lists some errors regarding mysql and posgresql support even though I already disabled them. There are a bunch of other errors that I don't understand what is their meanings at all. So I think I would better gave up :smile:
